core/mixins/_media-queries.scss

```diff
@@ -97,3 +105,16 @@
     @content;
   }
 }
+
+
+/**
+ * Retina.
+ */
+
+@mixin respond-retina($resolution: $retina-resolution) {
+  $resolution: if(unitless($resolution), $resolution * 96dpi, $resolution);
+  
+  @media (min-resolution: $resolution) {
+    @content;
+  }
+}
```

Comment:
So I gave this a spin and I'd expect to not have to pass in a parameter when I just want to use the Scally default (`$retina-resolution`) e.g.

```scss
@include respond-retina() {
  .foo {background-size: 100px auto;}
}
```

But that throws an compilation error.